tools/chrome-debug-extension/src/telemetryViewerPopup.tsx

```diff
@@ -17,11 +18,14 @@ type AppPhase =
     | "ConfigurationLoadFailed";
 
 const configurationTypeStorageKey = "configurationType";
+const defaultVersion = "#version#";
 
 export const TelemetryViewerPopup = (): React.ReactElement => {
     const [appPhase, setAppPhase] = React.useState<AppPhase>("Startup");
     const [session, setSession] = React.useState<Session | undefined>(undefined);
     const [configurationType, setConfigurationType] = React.useState<ConfigurationType>(undefined);
+    let newAvailableVersion: string;
+    // let newVersionDownload: string;
```

Comment:
Was originally planning on having some sort of dynamic update -- but that is more work so this is really a placeholder